app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/upload/UploadSuccessEmailScreen.kt

```diff
@@ -0,0 +1,108 @@
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.swisstransfer.ui.screen.newtransfer.upload
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import com.infomaniak.swisstransfer.R
+import com.infomaniak.swisstransfer.ui.components.*
+import com.infomaniak.swisstransfer.ui.images.AppImages.AppIllus
+import com.infomaniak.swisstransfer.ui.images.illus.uploadSuccessEmail.UploadSuccessEmail
+import com.infomaniak.swisstransfer.ui.theme.Margin
+import com.infomaniak.swisstransfer.ui.theme.SwissTransferTheme
+import com.infomaniak.swisstransfer.ui.utils.PreviewAllWindows
+
+// TODO: Use correct emails instead of hard-coded values
+@Composable
+fun UploadSuccessEmailScreen(
+    emails: List<String> = listOf(
+        "email@example.com",
+        "firstname.lastname@example.com",
+        "email@subdomain.example.com",
+        "firstname+lastname@example.com",
+        "email@123.123.123.123",
+        "email@[123.123.123.123]",
+        "\"email\"@example.com",
+        "1234567890@example.com",
+        "email@example-one.com",
+        "_______@example.com",
+        "email@example.name",
+        "email@example.museum",
+        "email@example.co.jp",
+        "firstname-lastname@example.com",
+    ),
+) {
+    BottomStickyButtonScaffold(
+        topBar = { BrandTopAppBar() },
+        bottomButton = {
+            LargeButton(
+                modifier = it,
+                titleRes = R.string.buttonFinished,
+                onClick = { /* TODO */ },
+            )
+        },
+        content = { Content(emails) },
+    )
+}
+
+@OptIn(ExperimentalLayoutApi::class)
+@Composable
+private fun Content(emails: List<String>) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .verticalScroll(rememberScrollState()),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Center,
+    ) {
+
+        IllustratedMessageBlock(
+            icon = AppIllus.UploadSuccessEmail.image(),
+            title = R.string.uploadSuccessEmailTitle,
+            description = R.string.uploadSuccessEmailDescription,
+            modifier = Modifier.padding(horizontal = Margin.Medium),
+        )
+
+        FlowRow(
+            modifier = Modifier.padding(Margin.Medium),
+            horizontalArrangement = Arrangement.Center,
+        ) {
+            emails.forEach {
+                EmailAddressChip(
+                    text = it,
+                    modifier = Modifier.padding(horizontal = Margin.XSmall)
+                )
+            }
+        }
+    }
+}
```

Comment:
I think we should put the horizontal padding on the parent column and have a spacer to separate the IllustratedMessageBlock from the FlowRow instead of specifying padding on both IllustratedMessageBlock and FlowRow